packages/ui/src/components/ProTable/index.tsx

```diff
@@ -194,6 +194,7 @@ const ProTable = <RecordType extends object & { key: string } = any>({
             customExtra = customExtra.concat(
                 <Tooltip title={dictionary.tooltips?.tableExport || 'Export as TSV'}>
                     <Button
+                        disabled={(headerConfig.itemCount?.total || 0) === 0}
```

Comment:
is it really this package responsibility to disable it ? or should it come from the app using the library ?

Reply:
unless disabled state makes sense for another reason other then "no data", I think the package can take care of it

Reply:
added headerConfig property